scripts/deploy.ts

```diff
@@ -54,6 +55,7 @@ function getDefaultGenesisRoot(parameters: DeploymentParameters) {
 
 async function main() {
     validateArgv();
+    await mcl.init();
```

Comment:
:facepalm: Don't know how I was getting this script to work before with pubkey creation but should work fine now.